include/xively_types.h

```diff
@@ -148,6 +149,92 @@ typedef void( xi_user_subscription_callback_t )( xi_context_handle_t in_context_
                                                  xi_state_t state,
                                                  void* user_data );
 
+/**
+ * @name  xi_sft_on_file_downloaded_callback_t
+ * @brief At a the end of a Secure File Transfer (SFT) HTTP file download the application
+ * should report back the result to the Xively C Client through such typed function.
+ *
+ * The application calls this function reporting the result of the SFT file HTTP download.
+ * This function is the trigger for continuing the update package download inside the
+ * Xively C Client. The application receives this function pointer during a request for
+ * HTTP download, in the application callback (`xi_sft_url_handler_callback_t` see below).
+ *
+ * @param [in] callback_data the data pointer received as attribute `callback_data` before
+ * in the call of HTTP downloader function `xi_sft_url_handler_callback_t`. The
+ * application should not do anything with this pointer *just* pass it here back.
+ * @param [in] filename the name of the file got downloaded and stored on the non-volatile
```

Comment:
the name of the file that was downloaded and stored...

Reply:
done